docs/features/asc1/stateful/hello_world.md

```diff
@@ -147,13 +148,13 @@ The _clear state program_ is used by the application to clear the _global_ and _
 
 Create a new file named `clear_state_program.teal` and add the following code:
 ```teal
-#pragma version 2
+#pragma version 4
 // This program clears program state
 
 int 1
 ```
 
-- `#pragma version 2` instructs the TEAL interpreter to use "version 2" OpCodes during execution. This is required for stateful smart contract applications
+- `#pragma version 4` instructs the AVM's TEAL interpreter to use "version 4" OpCodes during execution. This is required for stateful smart contract applications
```

Comment:
Should this be "instructs the AVM to use..."? This is similar to above comment. Let's just choose one and be consistent.